docs/source/security/cors.mdx

```diff
@@ -84,108 +82,135 @@ If you create a public API or an API to embed in websites you don't control your
 
 #### None of the above
 
-If your application doesn't fit into any of the above categories, the default Apollo Server CORS behavior should suit your use case. You can always specify origins in your CORS configuration later or disable cross-origin requests by [disabling CORS for your server](../api/apollo-server#cors).
+If your application doesn't fit into any of the above categories, the default Apollo Server CORS behavior should suit your use case. You can always specify origins in your CORS configuration later or disable cross-origin requests by disabling CORS for your server.
```

Comment:
not really the default

Reply:
I'm going across the board to address all of the "default" claims. Here, I replaced this with:

> If your application doesn't fit into any of the above categories, [`startStandaloneServer`](../migration#migrate-from-apollo-server)'s CORS behavior should suit your use case. You can always choose to swap to another Apollo Server integration later to [customize your CORS configuration](#configuring-cors-options-for-apollo-server).

(plus a todo to replace the migration link)